### DISCUSSION TITLE
验证码处理失败

### DISCUSSION THREAD

**Discussion by beggerlove** [IceTiki/ruoli-sign-optimization#146] [Q&A]
![Screenshot_2022-11-13-10-44-16-72_fab32b241c9ab51b0e057c943b753947.jpg](https://user-images.githubusercontent.com/65286493/201503109-42811f64-0d1e-4015-bff8-b18d2d1e2a36.jpg)

**Comment by IceTiki** [comment]
青龙面板还要额外传另一个参数，请看文档。

**Comment by IceTiki** [comment]
要加一个环境变量

**Comment by beggerlove** [comment]
> 要加一个环境变量

这个变量说实话  佬  我找了一上午  没任何头绪  能带带弟弟嘛  哭死.jpg

**Comment by beggerlove** [comment]
> 要加一个环境变量

![Screenshot_2022-11-13-17-32-42-57_fab32b241c9ab51b0e057c943b753947.jpg](https://user-images.githubusercontent.com/65286493/201515235-c8651fd1-5e54-4127-a06a-fda40b933e2a.jpg)大佬再给点儿提示吧

**Comment by beggerlove** [comment]
大佬有群聊可以加一下嘛

**Comment by IceTiki** [comment]
说实话这个是没有办法给提示的，本身就是用来筛人的(因为用的人太多今日校园就会加强反脚本
